include/distconfig.php

```diff
@@ -410,6 +410,18 @@
 
 // }}}
 
+// {{{ Mardown Render
+
+// Uncomment this line if you want to enable Mardown Rendering of README.md file in the path. 
+// You will need the Parsedown.php library for this to work. You can either manually add this 
+// library to include path or use the composer to install the same.
+// This will look for REAMDME.md file on the path and render it.
```

Comment:
Spelling: `REAMDME.md` vs. `README.md`

Suggested improvement:

> This will look for README.md file on the currently navigated path and render it.

Reply:
Done.